unattended_installer/cert_tool/certFunctions.sh

```diff
@@ -160,7 +163,7 @@ function cert_generateRootCAcertificate() {
     common_logger -d "Creating the root certificate."
 
     eval "openssl req -x509 -new -nodes -newkey rsa:2048 -keyout /tmp/wazuh-certificates/root-ca.key -out /tmp/wazuh-certificates/root-ca.pem -batch -subj '/OU=Wazuh/O=Wazuh/L=California/' -days 3650 ${debug}"
-
+    cert_setpermisions
```

Comment:
I would change the function call to this place:

https://github.com/wazuh/wazuh-packages/blob/2e1630dd0290742ef43149b5dae2ec90c25d6711/unattended_installer/cert_tool/certMain.sh#L181-L185

Reply:
Done [ebf6847ac4b7d6ddb4a7e469ed570c4ddf3c9f41](https://github.com/wazuh/wazuh-packages/pull/1415/commits/ebf6847ac4b7d6ddb4a7e469ed570c4ddf3c9f41)